src/test/java/com/wikia/webdriver/common/core/api/EditToken.java

```diff
@@ -22,31 +24,48 @@
 import com.wikia.webdriver.common.core.url.UrlBuilder;
 import com.wikia.webdriver.common.logging.PageObjectLogging;
 
-import java.util.Iterator;
-
 public class EditToken {
-  private String baseURL = new UrlBuilder().getUrlForWiki(Configuration.getWikiName())
-      + "/api.php";
-
   private static String EDIT_TOKEN_ERROR_MESSAGE = "Problem with edit token API call";
-
+  private String baseURL = new UrlBuilder().getUrlForWiki(Configuration.getWikiName()) + "/api.php";
   private User user;
 
   public EditToken(User user) {
     this.user = user;
   }
 
+  private static ResponseHandler<String> extractEditToken() {
+    return res -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this lambda expression number of lines from 21 to at most 20. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1188)